sharktank/sharktank/models/grok/grok.py

```diff
@@ -122,33 +123,32 @@ def prefill(
         self._assert_device(seq_block_ids)
         self._assert_device(*cache_state, dtype=self.activation_dtype)
         h = self.token_embedding(tokens)
-        h *= 78.38367176906169
+        h *= math.sqrt(h.shape[-1])
```

Comment:
This requires `import math`

Reply:
better as torch? I don't think math traces well

Reply:
Do we not have prefill here testing export? If not that needs to be added

Reply:
We should try to make faked versions of each model that is locally exportable. @KyleHerndon can you look into making faked `theta` parameters for the model? It can just be a single layer with smallers tensors.

Reply:
[Here](https://github.com/nod-ai/SHARK-Platform/blob/main/sharktank/sharktank/models/llama/testing.py) are the theta generators for attention/FFN/MOE blocks. We already have attention and prefill tests, might need to add export component in there.